kernelci/cli.py

```diff
@@ -262,6 +262,12 @@ class Args(object):
         'action': 'store_true'
     }
 
+    data_file = {
+        'name': '--data-file',
+        'help': "Path to the file with data to be submitted to storage",
+        'required': False
```

Comment:
If it's required, than this line should be dropped.

Reply:
Nice catch. Corrected.